docs/REQUIREMENTS.md

```diff
@@ -494,7 +503,7 @@ INGRESS_NGINX_CONFIG:
   controller:
     service:
       externalTrafficPolicy: Cluster
-      # loadBalancerIP: # Optional : Assigns a static IP to the SAS Viya ingress controller
+      # loadBalancerIP: <your static ip> # Assigns a specific IP for your loadBalancer
```

Comment:
There was a conflict in this line of the doc since it was also changed in the main branch.
When resolving the conflict, I opted for using the change from the staging branch since it was more recent.